server/migrations/database.js

```diff
@@ -23,7 +23,7 @@ const pool = new Pool({
 const userDetails = `
 DROP TABLE IF EXISTS userDetails cascade;
```

Comment:
Don't drop the table first

`CREATE TABLE IF NOT EXISTS` is fine

Reply:
ok